test/unit/protractor.flake.config.js

```diff
@@ -0,0 +1,20 @@
+// An example configuration file
+exports.config = {
```

Comment:
What about moving this to `test/unit/support` or `test/unit/support/fixtures`?

Reply:
Yes sir!